src/com/owncloud/android/lib/resources/users/RemoteGetUserQuotaOperation.java

```diff
@@ -0,0 +1,141 @@
+ *   copies of the Software, and to permit persons to whom the Software is
+ *   furnished to do so, subject to the following conditions:
+ *
+ *   The above copyright notice and this permission notice shall be included in
+ *   all copies or substantial portions of the Software.
+ *
+ *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *   EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ *   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *   NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ *   BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ *   ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ *   CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ *
+ */
+
+package com.owncloud.android.lib.resources.users;
+
+import com.owncloud.android.lib.common.OwnCloudBasicCredentials;
+import com.owncloud.android.lib.common.OwnCloudClient;
+import com.owncloud.android.lib.common.operations.RemoteOperation;
+import com.owncloud.android.lib.common.operations.RemoteOperationResult;
+import com.owncloud.android.lib.common.utils.Log_OC;
+
+import org.apache.commons.httpclient.NameValuePair;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.http.HttpStatus;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * @author marcello
+ */
+public class RemoteGetUserQuotaOperation extends RemoteOperation {
+
+    static public class Quota {
+        long mFree, mUsed, mTotal;
+        double mRelative;
+
+        public Quota(long free, long used, long total, double relative) {
+            mFree = free;
+            mUsed = used;
+            mTotal = total;
+            mRelative = relative;
+        }
+
+        public long getFree() { return mFree; }
+        public long getUsed() { return mUsed; }
+        public long getTotal() { return mTotal; }
+        public double getRelative() { return mRelative; }
+    }
+
+    private static final String TAG = RemoteGetUserQuotaOperation.class.getSimpleName();
+
+    private static final String NODE_OCS = "ocs";
+    private static final String NODE_DATA = "data";
+    private static final String NODE_QUOTA = "quota";
+    private static final String NODE_QUOTA_FREE = "free";
+    private static final String NODE_QUOTA_USED = "used";
+    private static final String NODE_QUOTA_TOTAL = "total";
+    private static final String NODE_QUOTA_RELATIVE = "relative";
+
+    // OCS Route
+    private static final String OCS_ROUTE ="/ocs/v1.php/cloud/users/";
+
+    @Override
+    protected RemoteOperationResult run(OwnCloudClient client) {
+        RemoteOperationResult result = null;
+        int status;
+        GetMethod get = null;
+
+
+        //Get the user
+        try {
+            OwnCloudBasicCredentials credentials = (OwnCloudBasicCredentials) client.getCredentials();
```

Comment:
Right now username may be accessed calling ```OwnCloudCredentials.getUsername()```, the casting is not needed anymore.

@przybylski , would you have the time to update it?

The rest seems OK.